cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -54,6 +54,15 @@ spec:
         # (Optional) Rook Agent toleration key. Set this to the key of the taint you want to tolerate
         # - name: AGENT_TOLERATION_KEY
         #   value: "<KeyOfTheTaintToTolerate>"
+        # (Optional) Rook Agent NodeAffinity Label Key. 
+        # Set this to the key of the Node Label to match.
+        # - name: AGENT_NODE_LABEL_KEY
```

Comment:
What if we combined the two variables into one where we use the `=` and `,` as delimeters? For example:
``` 
  - name: AGENT_NODE_AFFINITY
    value: "mylabel=valA,valB"
```
It would match more closely if they are using the label for `kubectl` queries, but maybe that doesn't matter.

Reply:
This sounds better than having separate variables. But,
1. How do you want to handle multiple labels?
2. Why aren't we setting affinity via crd?

Reply:
1. If we want to support multiple labels, I'm thinking we need another delimeter such as a colon or semicolon. But I'm also wondering if it's worth supporting. Maybe we should just start simple and support one label. I don't have a strong opinion either way.
1. Good question. The operator can manage multiple ceph clusters, so the settings for the operator, agent, and discovery pods are all independent of the CRDs. Their settings can only be specified in the operator deployment. If we can find a more natural way to pass these settings than an environment variable it would certainly be an improvement.

Reply:
1. Using commas sounds good as a separator. At least label selectors are separated like that. Maybe you could then use the label selector parse functions to prevent issues with commans in, e.g.: `my-cool-label-1="foo,bar",my-other-cool-label-2=foobar`

Reply:
To be honest, I think it's a bug *not* to support multiple labels/tolerations/taints/affinities/etc. While it's harder for us to implement, I would be shocked to find that most users only use a single one of any of these.

Reply:
I have updated the PR to support *multiple labels*, *multiple values per label key*, only *existence of label key*
Different labels are separated by `;`
@BlaineEXE @galexrt